Source/ACE.Entity/Character.cs

```diff
@@ -183,6 +183,21 @@ public Character(uint id, uint accountId)
         }
 
         /// <summary>
+        /// Sets the skill to trained status for a character
+        /// </summary>
+        /// <param name="skill"></param>
+        public bool TrainSkill(Skill skill, uint creditsSpent)
+        {
+            if(Skills[skill].Status != SkillStatus.Trained && Skills[skill].Status != SkillStatus.Specialized)
+                if(PropertiesInt[PropertyInt.AvailableSkillCredits] >= creditsSpent) { 
```

Comment:
again, not going to fail a PR over it, but we use C# curly brace formatting over Java/Javascript (curly braces always go on the next line).